packages/roosterjs-editor-core/lib/coreApi/createPasteFragment.ts

```diff
@@ -51,19 +52,12 @@ export const createPasteFragment: CreatePasteFragment = (
     const document = core.contentDiv.ownerDocument;
     let doc: HTMLDocument;
 
-    // Step 2: Fill the BeforePasteEvent object, especially the fragment for paste
-    if (!pasteAsText && !text && imageDataUri) {
-        // Paste image
-        const img = document.createElement('img');
-        img.style.maxWidth = '100%';
-        img.src = imageDataUri;
-        fragment.appendChild(img);
-    } else if (
+    // Paste HTML
+    if (
         !pasteAsText &&
```

Comment:
I think we don't need to check pasteAsText here. We can retrieve the meta tags as long as there is rawHtml